common/network_manager.go

```diff
@@ -338,33 +344,35 @@ func (m *NetworkManager) onRequest(event *network.EventRequestWillBeSent, interc
 		frame = m.frameManager.getFrameByID(event.FrameID)
 	}
 	if frame == nil {
-		debugLog("<networkmanager:onRequest> frame is nil")
+		m.logger.Debugf("NetworkManager", "frame is nil")
 	}
 
-	req := NewRequest(m.ctx, event, frame, redirectChain, interceptionID, m.userReqInterceptionEnabled)
+	req, err := NewRequest(m.ctx, event, frame, redirectChain, interceptionID, m.userReqInterceptionEnabled)
+	if err != nil {
+		m.logger.Errorf("NetworkManager", "error creating Request: %s", err)
```

Comment:
It's clear that it's an error. What about:

```golang
m.logger.Errorf("NetworkManager", "cannot create Request: %s", err)
```

Reply:
Sure, done in 7dc6ef7.